package.json

```diff
@@ -17,6 +17,7 @@
     "node": ">=6.0.0"
   },
   "devDependencies": {
+    "eslint": "4.11.0",
```

Comment:
We already had eslint through "fluid-grunt-eslint" did we need to pull it in here as well?

Reply:
Ah, I'll have to check that, as it wasn't showing up in my IDE's list of available modules.

Reply:
Done.